drivers/media/i2c/imx296.c

```diff
@@ -970,6 +975,8 @@ static int imx296_identify_model(struct imx296 *sensor)
 		return ret;
 	}
 
+	usleep_range(IMX296_STANDBY_DELAY, IMX296_STANDBY_DELAY);
```

Comment:
That's a very tight usleep when min and max are identical.
Why do we need to delay here anyway? Any chance of verifying Laurent's comment about needing to be out of standby?

Reply:
I'll put 2x for the max like the others.
We do need to come out of standby with a delay to read the id register, otherwise the I2C does fail. From what I can tell, this is because the register is in a different chip bank that is powered down in standby.